src/Services/Passenger/src/Passenger/Passengers/Models/ValueObjects/Name.cs

```diff
@@ -0,0 +1,18 @@
+namespace Passenger.Passengers.Models.ValueObjects;
+using System;
+
+public record Name : GenericValueObject<string>
```

Comment:
Please remove generic approach for this value object.

Reply:
done